src/components/Image/Image.tsx

```diff
@@ -13,7 +14,7 @@ class Image extends UIComponent<any, any> {
 
   static displayName = 'Image'
 
-  static handledProps = ['as', 'avatar', 'circular', 'className']
+  static handledProps = ['accBehavior', 'as', 'avatar', 'circular', 'className']
```

Comment:
Please avoid abbreviations in JS.  The HTML attr abbreviations are OK.  This is for consistency and clarity.  Minifiers handle making things small for us.